### PR TITLE
docs: Raw Printing

### PR DESCRIPTION
**Pull request by karthikeyan5**
Documentation for Raw Printing
screencapture-erpnext-com-local-8000-docs-user-manual-en-setting-up-print-raw-printing-2019-04-26-13_38_47

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/setting-up/print/index.txt [hunk 1]
```diff
@@ -6,4 +6,5 @@ letter-head
 address-template
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/print/index.txt]: add links to index.md not index.txt. do a pull to see new structure

File: erpnext_com/www/docs/user/manual/en/setting-up/print/print-settings.md [hunk 2]
```diff
@@ -1,13 +1,10 @@
 # Print Settings
 
-In Print Settings you can set your default printing preferences like Paper Size, default text size, whether you want to output as PDF or HTML etc.
-
-To edit print settings, go to:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/print/print-settings.md]: line 3, 2nd sentence should be 1. How to edit print settings. refer any doc from CRM, selling, buying, or stock to see new structure

File: erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# Raw Printing
+
+> Introduced in Version 12
+
+### 1. Introduction to Raw Printing
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md]: remove line 5 heading

File: erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+Sending a string of commands to a printer directly in its native language is called Raw Printing. Many thermal printers need these raw commands sent to them in order to perform functions like barcode printing, receipt printing, label printing, etc. Raw Printing bypasses the printer's drivers in most cases, making it very fast and reliable. Raw Printing is also capable of doing some advanced features such as cutting receipt paper, kicking out cash drawers, etc.
+
+### 2. Setting up Raw Printing in ERPNext
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md]: so now line 9 will be "1.", make subsequent changes

File: erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+![Raw Commands Print Format]({{docs_base_url}}/assets/img/setup/print/raw-command-print-format.png)
+
+Currently, any string-based printer languages can be used in the `Raw Commands` field in the print format. Writing raw commands requires knowledge of the printer's native language provided by the printer manufacturer. Please refer to the developer manual provided by the printer manufacturer on how to write their native commands.
+
+#### 2.3 Enable Raw Printing in the Print Setting
+
+To enable raw printing:
+1. Go to: **Settings > Printing > Print Settings > Raw Printing**.
+2. Check the `Enable Raw Printing` option.
+3. Save the settings.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md]: good steps. change step 4. to just "Save."

File: erpnext_com/www/docs/user/manual/en/setting-up/print/print-settings.md (hunk 2 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/print/print-settings.md]: I have followed this https://erpnext.com/docs/user/manual/en/buying/buying-settings. 

this seems to be the applicable structure. Also, I have now added numbering for the features.

File: erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md (hunk 3 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md]: done...

File: erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md (hunk 4 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md]: done...

File: erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md (hunk 5 above)
Comment (karthikeyan5) [reply erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md]: Done.

**Comment by karthikeyan5** [comment]
@prasadarr I have made the requested changes and updated the description with a screenshot.

**Pull request merged**